superset-frontend/src/dashboard/util/permissionUtils.ts

```diff
@@ -25,16 +25,14 @@ import {
 import { Dashboard } from 'src/types/Dashboard';
 import { findPermission } from 'src/utils/findPermission';
 
-// this should really be a config value,
-// but is hardcoded in backend logic already, so...
-const ADMIN_ROLE_NAME = 'admin';
+const ADMIN_ROLE_NAME = window.adminRole || 'Admin';
```

Comment:
```suggestion
const ADMIN_ROLE_NAME = window.adminRole;
```

Reply:
See https://github.com/apache/superset/pull/27779/files#r1544723819 as to why this will _always_ be set.